src/main/java/com/contentful/java/cma/ModulePersonalAccessTokens.java

```diff
@@ -39,7 +39,7 @@
    * @param callbackExecutor to tell on which thread it should run.
    */
   public ModulePersonalAccessTokens(Retrofit retrofit, Executor callbackExecutor) {
-    super(retrofit, callbackExecutor);
+    super(retrofit, callbackExecutor, null, null);
```

Comment:
Same argument as for the organisations: This endpoint only handles your own tokens, so there is no dependency to the space or environment.

I am thinking on whether I would expect me as a user to run into this: I configure a space and an environment, would I expect a personal access token to be only valid on the configured environment? Esp. since I am setting a scope on this token, I would expect the scope to carry the information....

So for now I would tend for not throwing an exception if one of the two (space, environment) is thrown ...